src/connection/ConnectionOptionsReader.ts

```diff
@@ -50,7 +55,7 @@ export class ConnectionOptionsReader {
      */
     async get(name: string): Promise<ConnectionOptions> {
         const allOptions = await this.all();
-        const targetOptions = allOptions.find(options => options.name === name || (name === "default" && !options.name));
```

Comment:
Can we instead update & add the `options` check here to verify that `options` is truthy instead of create a new static function?